cmake/OpenVDBHoudiniSetup.cmake

```diff
@@ -329,3 +329,50 @@ else()
   # Anticipated ABI version for H18
   set(OPENVDB_HOUDINI_ABI 6)
 endif()
+
+# ------------------------------------------------------------------------
+#  Configure GCC CXX11 ABI
+# ------------------------------------------------------------------------
+
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  if((CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL 5.1) OR
+     (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 5.1))
+    message(STATUS "GCC >= 5.1 detected. Configuring GCC CXX11 ABI for Houdini compatibility...")
+
+    execute_process(COMMAND echo "#include" " " "<string>"
```

Comment:
Curious, does this work with simply `echo "#include <string>"` instead of separate arguments?  I tested with cmake 3.13.4 and it worked there.

Reply:
I think you're right, I'll change this - thanks